src/main/java/org/bytecodeparser/instruction/PrettyPrintTypes.java

```diff
@@ -0,0 +1,42 @@
+package org.bytecodeparser.instruction;
+
+import org.bytecodeparser.utility.ThrowingFunction;
+
+import java.io.DataInputStream;
+import java.io.IOException;
+import java.util.function.Function;
+
+public enum PrettyPrintTypes {
```

Comment:
why class with name PrettyPrintTypes do something with DataInoutStream?

Reply:
PrettyPrintTypes describes the logic of working with DataInoutStream and o nothing with it directly.

... or I don't get your point...

Reply:
I mean it is confusing naming. PrettyPrintTypes supposed to describe different types of printing somethin pretty, but it also does dataInputStream manipulation. Maybe smth like InstructionArgumentResolver

Reply:
will do it with follow up commit by myself